docs/src/install/gettingstarted.rst

```diff
@@ -0,0 +1,149 @@
+***************
+Getting Started
+***************
+
+Welcome! In this chapter, we give a quick overview of Oríon's main features and how it can help you
+streamline your machine learning workflow whether you are a researcher or engineer.
+
+Oríon is a black box function optimization library with a key focus on usability and integrability
+for its users. For example, as an machine learning engineer, you can integrate Oríon to your
+existing ML workflow to handle reliably the hyperparameter optimization and tuning phase. As a ML
+researcher, you can use Oríon to tune your models but also integrate your own algorithms to Oríon to
+serve as an efficient optimization engine and compare with other algorithms in the same context and
+conditions.
+
+Oríon is built to be non-intrusive, highly integrable, operate in parallel environments, and produce
+reproducible experiments. It takes only one line of code to start using it in your existing
```

Comment:
Maybe at 'if your script support hyperparameter definition through commandline arguments of configuration files, then it takes only one line of code ...'.

Reply:
This is a good suggestion but I feel it sounds too specific for an introduction.